src/target/mod.rs

```diff
@@ -597,6 +597,12 @@ pub trait Target {
         true
     }
 
+    /// Support for register info query
```

Comment:
i'd just copy over the comment from `use_target_description_xml` wholesale, swapping out the methods with those associated with `qRegisterInfo`